src/workerd/jsg/buffersource.h

```diff
@@ -76,7 +76,7 @@ using BufferSourceViewConstructor = v8::Local<v8::Value> (*)(Lock&, BackingStore
 class BackingStore {
  public:
   template <BufferSourceType T = v8::Uint8Array>
-  static BackingStore from(kj::Array<kj::byte> data) {
+  static BackingStore from(Lock& js, kj::Array<kj::byte> data) {
```

Comment:
BackingStore::from() needing a lock looks unrelated to the IsolateGroup changes, can you add the rationale in a comment or a commit description?

Reply:
I added a comment to explain.  This is because the ArrayBuffer backing needs to be allocated in the correct
sandbox, so we need to make the isolate explicit here.